frontend/src/metabase/lib/utils.js

```diff
@@ -135,9 +135,15 @@ const MetabaseUtils = {
     return JSON.parse(JSON.stringify(a));
   },
 
-  // this should correctly compare all version formats Metabase uses, e.x.
-  // 0.0.9, 0.0.10-snapshot, 0.0.10-alpha1, 0.0.10-rc1, 0.0.10-rc2, 0.0.10-rc10
-  // 0.0.10, 0.1.0, 0.2.0, 0.10.0, 1.1.0
+  /**
+   * this should correctly compare all version formats Metabase uses, e.x.
+   * 0.0.9, 0.0.10-snapshot, 0.0.10-alpha1, 0.0.10-rc1, 0.0.10-rc2, 0.0.10-rc10
+   * 0.0.10, 0.1.0, 0.2.0, 0.10.0, 1.1.0
+   * @type {{
```

Comment:
this is a handy way of typing something in a js file, i need this specific overload because otherwise when used in a .sort call ts would complain about the null return